ui/imports/shared/controls/SlippageSelector.qml

```diff
@@ -2,24 +2,29 @@ import QtQuick 2.15
 import QtQuick.Controls 2.15
 import QtQuick.Layouts 1.15
 
+import StatusQ.Core 0.1
 import StatusQ.Controls 0.1
+import StatusQ.Core.Theme 0.1
 
 Control {
     id: root
 
     property double value: d.defaultValue
     readonly property double defaultValue: d.defaultValue
-    readonly property bool valid: customInput.activeFocus && customInput.valid
-                                  || buttons.value !== null
+    readonly property bool valid: (customInput.activeFocus && customInput.valid) || buttons.value !== null
     readonly property bool isEdited: root.value !== d.defaultValue
 
+    property double valueTooHighThreshold: 5
```

Comment:
nitpick: Maybe `maxValue`? 😄

Reply:
Nah... this is really when we just issue the warning; as opposed to the hard maximum (which is 100%)